### PR TITLE
Issue #45 create CarModelEntity

### LINKED ISSUE #45: Create CarModelEntity

### PR DESCRIPTION
**Pull request by JustRevolt**

### CODE REVIEW

**Review by nikaalin — APPROVED** [review]

File: src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            return false;
+        }
+
+        CarModelEntity that = (CarModelEntity) o;
+
+        return (
+                id == that.id &&
+                brandName.equals(that.brandName) &&
+                modelName.equals(that.modelName) &&
+                pricePerHour.equals(that.pricePerHour)
```
Comment (a-deriabin) [inline src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java]: This will fail if brandName or modelName is null

**Review by a-deriabin — CHANGES_REQUESTED** [review]

File: src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java (hunk 1 above)
Comment (JustRevolt) [reply src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java]: Change

**Review by a-deriabin — APPROVED** [review]

**Review by ChirkinaTanya — APPROVED** [review]

File: src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        CarModelEntity that = (CarModelEntity) o;
+
+        return (
+                Objects.equals(id, that.id) &&
```
Comment (PavelEzhkov) [inline src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java]: Is int object in java? Use == for basic types and Objects.equals() for Objects

**Review by PavelEzhkov — COMMENTED** [review]
fix it

File: src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java (hunk 2 above)
Comment (JustRevolt) [reply src/main/java/com/ifmo/epampractice/entity/CarModelEntity.java]: @PavelEzhkov done

**Review by PavelEzhkov — APPROVED** [review]

**Pull request merged**